seqtk/seq conda tests fail

Follow up to #1405 

**Only the conda tests fail**, singularity and docker both pass so I'm guessing it's an error with the package version in conda.

```
=================================== FAILURES ===================================
_________________________________ test session _________________________________
Observed md5sum 'e07203757afaf3353ea9b72c80507469' not equal to expected md5sum '2f009f1647971a97b4edec726a99dc1a' for file '/home/runner/pytest_workflow_fa47kraz/seqtk_seq_test_seqtk_seq_fq/output/seqtk/test.seqtk-seq.fasta.gz'
```

--- a/tests/modules/nf-core/seqtk/seq/test.yml
+++ b/tests/modules/nf-core/seqtk/seq/test.yml
@@ -5,7 +5,8 @@
     - seqtk
   files:
     - path: output/seqtk/test.seqtk-seq.fasta.gz
-      md5sum: 50d73992c8c7e56dc095ef47ec52a754
+      contains:
+        - ">MT192765.1 Severe acute respiratory syndrome coronavirus 2 isolate SARS-CoV-2/human/USA/PC00101P/2020, complete genome"
 
 - name: seqtk seq test_seqtk_seq_fq
   command: nextflow run ./tests/modules/nf-core/seqtk/seq -entry test_seqtk_seq_fq -c ./tests/config/nextflow.config

